### PR TITLE
added option to change week and day in signs, and added translation for everything that had day, hour and week

### PR DESCRIPTION
**Pull request by thiagogebrimm**
added option to change week and day in signs, and added translation for everything that had day, hour and week
a small change in the pt-br translation

I think this helps a lot for users like me who don't use English as their default language

### CODE REVIEW

**Review by EtienneDx — COMMENTED** [review]
Looks good, I don't have much free time but happy to see the contribution! Any chance you can revert to the original formating? The changes seems mostly good but hard to make the diff since the formating changed :/

**Review by EtienneDx — COMMENTED** [review]
Can't wait to have time to get 100% back into it, for now looks like we need to move some stuff to StringBuilder x)
Otherwise all good, I'll merge in the next two weeks someday when I can get back to it. Thanks for the PR :)

**Comment by thiagogebrimm** [comment]
> Looks good, I don't have much free time but happy to see the contribution! Any chance you can revert to the original formating? The changes seems mostly good but hard to make the diff since the formating changed :/

there is no way to reverse it, I fixed the formatting using intellij as soon as I opened the project, I would have to redo everything :(

File: src/me/EtienneDx/RealEstate/REListener.java [hunk 1]
```diff
@@ -19,517 +20,446 @@
...
+
+                // all should be good, we can create the auction
+                event.setCancelled(true);
+                RealEstate.transactionsStore.auction(claim, player, price, event.getBlock().getLocation(), duration, bidStep);
+            }
+        }
+    }
+
+    private int parseDuration(String line) {
+        Pattern p = Pattern.compile("^(?:(?<" + RealEstate.instance.config.cfgDatesWeeks.toLowerCase() + ">\\d{1,2}) ?" + RealEstate.instance.config.cfgDatesWeeks.toLowerCase().charAt(0) + "(?:" + RealEstate.instance.config.cfgDatesWeeks.toLowerCase().substring(1, RealEstate.instance.config.cfgDatesWeeks.length()) + "?)?)? ?(?:(?<" + RealEstate.instance.config.cfgDatesDay.toLowerCase() + "s>\\d{1,2}) ?" + RealEstate.instance.config.cfgDatesDays.toLowerCase().charAt(0) + "(?:" + RealEstate.instance.config.cfgDatesDays.toLowerCase().substring(1, RealEstate.instance.config.cfgDatesDays.length()) + "?)?)?$", Pattern.CASE_INSENSITIVE);
```
[the remark this answers is not in the document]
Comment (thiagogebrimm) [reply src/me/EtienneDx/RealEstate/REListener.java:389]: ok

**Pull request closed without merging**